server/helpers/dateHelpers.js

```diff
@@ -46,3 +46,63 @@ export function isValidStartDate({ endDate = moment(), startDate = moment().subt
   // check if both date from and to are provided
   return { dateFrom, dateTo };
 }
+
+/**
+ * helper for the date Query
```

Comment:
Be more specific as to what the function is _helping_ to do.
Also, a bit more details of the other parameters will be more useful

Reply:
resolved